rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/sca/GetDependencyRiskDetailsResponse.java

```diff
@@ -72,10 +76,12 @@ public DependencyRiskDto.Type getType() {
     return type;
   }
 
+  @CheckForNull
   public String getVulnerabilityId() {
     return vulnerabilityId;
   }
 
+  @CheckForNull
```

Comment:
I would say changing the nullability is some kind of breaking change, maybe worth documenting it